backend/src/main/resources/graphql/main.graphqls

```diff
@@ -389,6 +389,7 @@ type Query {
     namePrefixMatch: String
     includeArchivedFacilities: Boolean = false
       @requiredPermissions(allOf: ["VIEW_ARCHIVED_FACILITIES"])
+    orgExternalId: String
```

Comment:
Should this field optional given that if the operation is used by a non site admin the organization comes from the orgContext?